azdev/params.py

```diff
@@ -100,6 +100,32 @@ def load_arguments(self, _):
                         'Defaults to "high".')
     # endregion
 
+    # region scan & mask
+    for scope in ['scan', 'mask']:
+        with ArgumentsContext(self, scope) as c:
+            c.argument('file_path', options_list=['--file-path', '-f'],
+                       help='Path of the file you want to scan secrets for')
+            c.argument('directory_path', options_list=['--directory-path', '-d'],
+                       help='Path of the folder you want to scan secrets for')
+            c.argument('recursive', options_list=['--recursive', '-r'],
+                       help='Scan the directory recursively')
+            c.argument('data', help='Raw string you want to scan secrets for')
+            c.argument('save_scan_result', options_list=['--save-scan_result', '--save'], type=bool,
+                       help='Whether to save scan result to file or not')
+            c.argument('scan_result_path', options_list=['--scan-result-path', '--result'],
```

Comment:
Is it necessary to provide more explanation on the relationship between the two options `--save-scan-result` and `--scan-result-path` to help users understand their purpose and relationship with each other, and inform users of the default path for saving?

Reply:
added detailed help msg